src/transformStateWithClones.js

```diff
@@ -8,6 +8,32 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  let previousState = { ...state };
```

Comment:
The variable name 'previousState' is not descriptive enough. It is better to use a name like 'stateCopy' to clearly indicate that this variable is a copy of the current state.